lib/ansible/config/base.yml

```diff
@@ -329,12 +329,20 @@ ACTION_WARNINGS:
   - {key: action_warnings, section: defaults}
   type: boolean
   version_added: "2.5"
+BECOME_PLUGIN_PATH:
```

Comment:
Most of the plugin path config items are of form 'DEFAULT_CACHE_PLUGIN_PATH' etc. 

If there isn't a particular reason for BECOME_PLUGIN_PATH instead of DEFAULT_BECOME_PLUGIN_PATH, may be better to keep it consistent with naming of other plugin path config items.

Reply:
i'm actually  moving away from DEFAULT since that is a misleading tag